.github/workflows/android-release.yml

```diff
@@ -44,6 +41,9 @@ jobs:
         run: npm install --ignore-scripts          
         working-directory: platform/android
 
+      - name: Install ccache
+        run: sudo apt-get install -y ccache
+
       - name: Prepare ccache
         run: ccache --clear       
```

Comment:
Nit: We can combine the steps here.

Reply:
Let's do that with the reusable ccache workflow.